content/departments/product-engineering/process/planning-communication.md

```diff
@@ -0,0 +1,80 @@
+# Product/Engineering planning communication
+
+This page documents how we communicate our plans and progress.
```

Comment:
Narrative strategy pages are missing from this section, but should be here IMHO as they are important to communicate what we're prioritizing and why, beyond the list of items in the roadmap. https://handbook.sourcegraph.com/strategy-goals/strategy#team-strategy-pages. Do you agree?

Reply:
Agree, I moved this here.